test/functional/helpers/report-helper.js

```diff
@@ -0,0 +1,63 @@
+function checkTestRelevanceByBrowserAlias (test, browserAlias) {

```

Comment:
You are filtering browsers not tests, so maybe they are what may be relevant or irrelevant?
`checkTestRelevanceByBrowserAlias` -> `isBrowserRelevant`